src/components/CategoryPage.svelte

```diff
@@ -1,33 +1,32 @@
 <script lang="ts">
   import { page } from "$app/stores";
-  import { mapStore, selectedGeographyStore } from "../stores/stores";
+  import { mapStore, selectedGeographyStore, vizStore } from "../stores/stores";
   import { fetchVizData } from "../data/fetchVizData";
-  import { fetchGeographyData } from "../data/fetchGeographyData";
-  import { getCodesForCategory, getSelectedGeography } from "../helpers/categoryHelpers";
+  import { getCodesForCategory } from "../helpers/categoryHelpers";
   import CensusTable from "./CensusTable.svelte";
   import NavigationComponent from "./NavigationComponent.svelte";
   import topics from "../data/content";
   import CategoryHeading from "../components/CategoryHeading.svelte";
 
-  $: variableData = $selectedGeographyStore?.variableData;
+  $: variableData = $vizStore?.variableData;
   $: params = $page.params;
   $: topicSlug = params.topic;
   $: topic = topics.find((t) => t.slug === topicSlug);
   $: variableSlug = params.variable;
   $: variable = topic.variables.find((v) => v.slug === variableSlug);
   $: search = $page.url.search;
-  $: selectedGeography = getSelectedGeography($page.url);
   $: selectedGeographyDisplayName = $selectedGeographyStore?.displayName;
+  $: selectedGeographyGeoCode = $selectedGeographyStore?.geoCode;
   $: categorySlug = params.category;
   $: category = variable.categories.find((c) => c.slug === categorySlug);
 
   $: if ($mapStore) {
     let codes = getCodesForCategory(params.topic, params.variable, params.classification, params.category);
-    fetchVizData({ ...codes, geoType: $mapStore.geoType, bbox: $mapStore.bbox });
-    fetchGeographyData({
-      totalCode: codes.totalCode,
-      categoryCodes: codes.categoryCodes,
-      geoCode: selectedGeography.geoCode,
+    fetchVizData({
+      ...codes,
+      geoType: $mapStore.geoType,
+      geoCode: selectedGeographyGeoCode,
```

Comment:
This looks odd - `geoType` and `geoCode` form a pair of values, but they are coming from different places that could be different. Maybe they should both come from  the `selectedGeographyStore`